### PR TITLE
Add PC-blend preheat preset

### PR DESCRIPTION
**Pull request by DRracer**
PFW-1127

### CODE REVIEW

File: Firmware/variants/1_75mm_MK2-RAMBo10a-E3Dv6full.h [hunk 1]
```diff
@@ -330,6 +330,10 @@ PREHEAT SETTINGS
 #define ASA_PREHEAT_HPB_TEMP 105
 #define ASA_PREHEAT_FAN_SPEED 0
 
+#define PC_PREHEAT_HOTEND_TEMP 275
+#define PC_PREHEAT_HPB_TEMP 105
+#define PC_PREHEAT_FAN_SPEED 0
```
Comment (3d-gussner) [inline Firmware/variants/1_75mm_MK2-RAMBo10a-E3Dv6full.h:335]: Why only here and not in all other variants?

**Review by mkbel — APPROVED** [review]

File: Firmware/ultralcd.cpp [hunk 2]
```diff
@@ -2555,6 +2561,7 @@ void lcd_generic_preheat_menu()
         MENU_ITEM_SUBMENU_P(PSTR("PLA  -  " STRINGIFY(PLA_PREHEAT_HOTEND_TEMP) "/" STRINGIFY(PLA_PREHEAT_HPB_TEMP)),mFilamentItem_PLA);
         MENU_ITEM_SUBMENU_P(PSTR("PET  -  " STRINGIFY(PET_PREHEAT_HOTEND_TEMP) "/" STRINGIFY(PET_PREHEAT_HPB_TEMP)),mFilamentItem_PET);
         MENU_ITEM_SUBMENU_P(PSTR("ASA  -  " STRINGIFY(ASA_PREHEAT_HOTEND_TEMP) "/" STRINGIFY(ASA_PREHEAT_HPB_TEMP)),mFilamentItem_ASA);
+        MENU_ITEM_SUBMENU_P(PSTR("PC   -  " STRINGIFY(PCB_PREHEAT_HOTEND_TEMP) "/" STRINGIFY(PCB_PREHEAT_HPB_TEMP)),mFilamentItem_PC);
```
Comment (3d-gussner) [inline Firmware/ultralcd.cpp:2564]: PC_PREHEAT ... auto correct made it PCB I guess

**Review by 3d-gussner — CHANGES_REQUESTED** [review]

File: Firmware/variants/1_75mm_MK2-RAMBo10a-E3Dv6full.h (hunk 1 above)
Comment (DRracer) [reply Firmware/variants/1_75mm_MK2-RAMBo10a-E3Dv6full.h:335]: the other variants do not have the fan like this (this is MK2)

File: Firmware/ultralcd.cpp (hunk 2 above)
Comment (DRracer) [reply Firmware/ultralcd.cpp:2564]: oh, you are right .... I hate #defines :(

File: Firmware/variants/1_75mm_MK2-RAMBo10a-E3Dv6full.h (hunk 1 above)
Comment (3d-gussner) [reply Firmware/variants/1_75mm_MK2-RAMBo10a-E3Dv6full.h:335]: And IT the `aaa_PREHEAT_FAN_SPEED` is nowhere used in the code.
So so it is very old.
BTW The MK2 RAMBo13a doesn't have that. guess someone forgot to cleanup the RAMBo10a

**Pull request merged**